content/manuals/desktop/features/kubernetes.md

```diff
@@ -55,29 +68,46 @@ $ kubectl config use-context docker-desktop
 
 > [!TIP]
 >
-> Run the `kubectl` command in a CMD or PowerShell terminal, otherwise `kubectl config get-contexts` may return an empty result. 
->
-> If you are using a different terminal and this happens, you can try setting the `kubeconfig` environment variable to the location of the `.kube/config` file. 
+> If the `kubectl` config get-contexts command returns an empty result, try:
+> 
+> - Running the command in a CMD or PowerShell terminal.
```

Comment:
nit: I don't think I have ever seen Command Prompt / cmd.exe referred to as `CMD`. Also, "PowerShell terminal" is inaccurate. Suggesting to simplify:

```suggestion
> - Running the command in the Command Prompt or PowerShell.
```